infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/utils/LogReplicationStreamTableManager.java

```diff
@@ -283,4 +318,10 @@ public Map<UUID, List<UUID>> getStreamingConfigOnSink() {
     public static Set<UUID> getMergeOnlyStreamIdList() {
         return MERGE_ONLY_STREAM_ID_LIST;
     }
+
+    private enum VersionResult {
+        INITIALIZING,
+        VERSION_CHANGING,
```

Comment:
VersionResult already has the word Version, and since you're trying to capture what is the result of comparing persisted Version with current/newly read version, maybe these capture it best:

UNSET
CHANGE
SAME